app/models/concerns/galtersufia/generic_file/license_data.txt

```diff
@@ -0,0 +1,77 @@
+{
+  'All Rights Reserved': {
```

Comment:
I need to ask about this, but I didn't think it should hold up opening a PR.

From what I understand this is a standard way of saying "No one can do anything with this item" and because of that there isn't a website, id, or really even a license so I'm not sure what goes here.

Reply:
Side note: I just copied over JSON directly from [SPDX's](https://spdx.org/licenses/) list except for the licenses I had to enter myself. Because of this there are a lot of unnecessary fields included, but I didn't think they were really hurting anything so I just left them.

Reply:
This was one I actually had some questions on still. As I understand it "All Rights Reserved" isn't really a license in the sense that it doesn't give ANY rights to anyone else to use it. It's basically a statement saying you can't do anything with this. That said, we can probably just add an identifier that we all agree on. I'm not sure what it would look like though. Maybe since the rest of the identifiers are based on acronyms we could use something like `ARR`?